src/status_im/contexts/wallet/send/select_address/tabs/view.cljs

```diff
@@ -17,15 +17,20 @@
         :description     (i18n/label :t/here-is-a-cat-in-a-box-instead)
         :image           (resources/get-themed-image :cat-in-box theme)
         :container-style style/empty-container-style}]
-      (into [rn/view {:style style/my-accounts-container}]
-            (map (fn [{:keys [color address] :as account}]
-                   [quo/account-item
-                    {:account-props (assoc account :customization-color color)
-                     :on-press      #(rf/dispatch [:wallet/select-send-address
-                                                   {:address   address
-                                                    :recipient account
-                                                    :stack-id  :screen/wallet.select-address}])}]))
-            other-accounts))))
+      [rn/view {:style style/my-accounts-container}
+       (doall (for [{:keys [color address] :as account} other-accounts]
```

Comment:
Tiny detail, normally `for` goes in the next line after `doall` to reduce indentation.

Reply:
Done